user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Query_Data_Sources/Get_views.md

```diff
@@ -5,3 +5,10 @@ uid: Get_views
 # Get views
 
 Available from DataMiner 10.2.0/10.1.4 onwards. The *Get views* data source retrieves a list of the views in the DMS. By default, only the columns *View ID* and *Name* are included, but you can include additional columns using a [*Select* operator](xref:GQI_Select).
+
+## Updates
+
+From DataMiner 10.3.11 onwards, this data source supports [real-time updates](xref:Query_updates).
```

Comment:
10.3.10 or 10.3.11?

Reply:
Same reason as mentioned before. I can change it to 10.3.10 if you prefer.